test/services/log-source-map-service.ts

```diff
@@ -0,0 +1,87 @@
+import { Yok } from "../../lib/common/yok";
+import { assert } from "chai";
+import * as path from "path";
+import { LogSourceMapService } from "../../lib/services/log-source-map-service";
+import { DevicePlatformsConstants } from "../../lib/common/mobile/device-platforms-constants";
+import { FileSystem } from "../../lib/common/file-system";
+import { stringReplaceAll } from "../../lib/common/helpers";
+
+function createTestInjector(): IInjector {
+	const testInjector = new Yok();
+	testInjector.register("projectDataService", {
+		getProjectData: () => {
+			return {
+				getAppDirectoryRelativePath: () => {
+					return "src";
+				},
+				projectIdentifiers: {
+					android: "org.nativescript.sourceMap",
+					ios: "org.nativescript.sourceMap"
+				}
+			};
+		}
+	});
+	testInjector.register("platformsDataService", {
+		getPlatformData: (platform: string) => {
+			return {
+				appDestinationDirectoryPath: path.join(__dirname, ".." , "files", "sourceMapBundle", platform.toLowerCase())
+			};
+		}
+	});
+	testInjector.register("fs", FileSystem);
+	testInjector.register("devicePlatformsConstants", DevicePlatformsConstants);
+	testInjector.register("logSourceMapService", LogSourceMapService);
+
+	return testInjector;
+}
+
+function toPlatformSep(filePath: string) {
+	return stringReplaceAll(filePath, "/", path.sep);
+}
+
+const testCases: IDictionary<Array<{caseName: string, message: string, expected: string}>> = {
+	"android": [{
+			caseName: "trace massage",
```

Comment:
not a merge stopper, but I assume you meant message not massage (or it is just Thursday 😸 )